.gitattributes

```diff
@@ -0,0 +1 @@
+test/output*/** linguist-vendored
```

Comment:
This should remove the html here in GitHub language breakdown:

![image](https://github.com/user-attachments/assets/a1a90633-7e65-4055-b81d-76436d248948)

Since the html is only test data